### ISSUE TITLE
purify() doesn't work on IE and Safari

### ISSUE THREAD

**Issue by dshorowitz** [KanoComputing/nodejs-profanity-util#3]
In the profanity.js file, line 79 fails on both Safari and IE if you pass in a string to the purify function. That's because Object.keys() in ES5 doesn't support passing in a string and those browsers haven't updated yet to the ES6 standard which does.

**Comment by skliffmueller** [comment]
I don't think this needs to be patched in the official version of profanity-utils. But since I'm working with an older version of node js on an application, I was forced to make a fix. My fork is at https://github.com/skliffmueller/nodejs-profanity-util if you are interested.

**Comment by tancredi** [comment]
Sorry for the late reply @dshorowitz - @skliffmueller just merged your fork and re-published the npm package. Thank you both.

**Comment by skliffmueller** [comment]
It works, problem solved!
